troubleshooting/ipad-multitasking-support-ios-publish-to-fail.md

```diff
@@ -0,0 +1,48 @@
+---
+title: Invalid Bundle Error When Publishing to the App Store
+description: "Publishing to the Apple App Store fails with ERROR ITMS-90474: Invalid Bundle. iPad Multitasking support requires these orientations."
+type: troubleshooting
+page_title: Publishing App with IPad Multitasking Support Requires All Device Orientations to Be Enabled
```

Comment:
iPad

Although the convention is to capitalize every major word in a title, in this case, the casing of the product must be preserved.